cmd/juju/cloud/list.go

```diff
@@ -53,8 +53,9 @@ var listCloudsDoc = "" +
 	"This command's default output format is 'tabular'. Use 'json' and 'yaml' for\n" +
 	"machine-readable output.\n" +
 	"\n" +
-	"Cloud metadata sometimes changes, e.g. providers add regions. Use the `update-clouds`\n" +
-	"command to update the current Juju client.\n" +
+	"Cloud metadata sometimes changes, e.g. providers add regions. Use the `update-public-clouds`\n" +
+	"command to update public clouds from a central location or `update-cloud` to" +
+	"update your customised clouds.\n" +
```

Comment:
... customised or on-prem ...
?

Reply:
'to update other clouds'? This would imply non-public which is kind of what we want to say here...